### PR TITLE
Add container subcommand, update readme

### PR DESCRIPTION
**Pull request by gravcat**

### CODE REVIEW

File: cmd/container.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// containerCmd represents the service command
```
Comment (BenOvermyer) [inline cmd/container.go]: This comment should refer to the container command, not the service command.

File: cmd/listContainers.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// listServicesCmd represents the listServices command
```
Comment (BenOvermyer) [inline cmd/listContainers.go]: This comment should be changed to refer to the listContainers command.

File: cmd/listContainers.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+		for _, e := range portainer.Endpoints {
+			printContainersForEndpoint(e)
+		}
+	},
+}
+
+func init() {
+	containerCmd.AddCommand(listContainersCmd)
+	//listServicesCmd.Flags().BoolVarP(&filterContainersBroken, "broken", "b", false, "Display only broken containers")
```
Comment (BenOvermyer) [inline cmd/listContainers.go]: If this line doesn't do anything it should be removed.

**Review by BenOvermyer — CHANGES_REQUESTED** [review]

File: cmd/listContainers.go (hunk 3 above)
Comment (gravcat) [reply cmd/listContainers.go]: Thanks -- done!

File: cmd/container.go (hunk 1 above)
Comment (gravcat) [reply cmd/container.go]: Done! You can probably tell what I used for a springboard.

File: cmd/listContainers.go (hunk 2 above)
Comment (gravcat) [reply cmd/listContainers.go]: Done.

File: cmd/container.go [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
```
Comment (AMC-7) [inline cmd/container.go:19]: Remove extra space.

File: cmd/containers.go [hunk 5]
```diff
@@ -20,13 +6,15 @@ import (
...
 
+
 // Container is a Docker container
 type Container struct {
 	ID    string
 	Image string
 	State string
 }
 
+
```
Comment (AMC-7) [inline cmd/containers.go:17]: Remove extra space.

**Review by AMC-7 — APPROVED** [review]
A couple of optional comments, approved nonetheless.

**Pull request merged**